app/frontend/src/views/Dashboard.vue

```diff
@@ -1,18 +1,47 @@
 <template>
   <v-container class="dashboard text-center">
     <h1>Performance Dashboard</h1>
-    <p>The dashboard is still under construction but will appear here soon</p>
-    <router-link :to="{ name: 'home' }">
-      <v-btn color="primary" class="home-btn" id="not-found-home" large>
-        <v-icon left>home</v-icon>
-        <span>Home</span>
-      </v-btn>
-    </router-link>
+    <div v-if="dashboardUrl" class="dashboard-container">
+      <iframe :src="dashboardUrl" />
+    </div>
+    <div v-else>
+      <p>No dashboard has been configured.</p>
+      <router-link :to="{ name: 'home' }">
+        <v-btn color="primary" class="home-btn" id="dashboard-home" large>
+          <v-icon left>home</v-icon>
+          <span>Home</span>
+        </v-btn>
+      </router-link>
+    </div>
   </v-container>
 </template>
 
 <script>
 export default {
-  name: 'dashboard'
+  name: 'dashboard',
+  computed: {
+    dashboardUrl() {
+      return this.$configService.get('dashboardUrl');
+    }
+  }
 };
 </script>
+
+<style lang="scss" scoped>
+.dashboard-container {
```

Comment:
do we need the iframe resizer js?

Reply:
I actually tried to integrate that in but wasn't able to get that to work with the vue.js framework (even their example on how to add it as a vue directive kept failing). I did the media-query approach as a makeshift workaround for the moment to make sure it roughly fits on the page.

Reply:
Does iframeresizer work on a Kibana dash?
It has to have a equivalent "client JS" in the target of the iframe that the resizer communicates to. (Metabase provides this in its dash)